src/lib/bmp/bmp.h

```diff
@@ -0,0 +1,47 @@
+/**
+  * @file bmp.h
+  * @brief lib for read and draw BMP
+  * @author Filipp Chubukov
+  * @version
+  * @date 03.07.2019
+*/
+
+#ifndef LIB_BMP_H_
+#define LIB_BMP_H_
+
+#define HEADER_END 53 /* position of header's end */
```

Comment:
Why declare it in this header? It's used only in bmp.c

Reply:
fixed